### PR TITLE
Update level edges

### PR DESCRIPTION
**Pull request by kdhageman**
Improves edge generation, see [here](https://github.com/ratenbuuren/ProjectBeard/pull/22) for previous style. You can use a slider to select the edge scale (from 0 to 1)

screen shot 2017-07-18 at 21 25 24

screen shot 2017-07-18 at 21 25 45

### CODE REVIEW

**Comment by ratenbuuren** [comment]
Without [sorting layers](https://docs.unity3d.com/Manual/class-TagManager.html):

screen shot 2017-07-19 at 17 14 16

**Comment by kdhageman** [comment]
> Without sorting layers:

Fixed

File: Assets/Scripts/LevelGenerator.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 using System;
+using System.Security.Cryptography.X509Certificates;
```
Comment (ratenbuuren) [inline Assets/Scripts/LevelGenerator.cs]: Not used?

File: Assets/Scripts/LevelGenerator.cs [hunk 2]
```diff
@@ -11,108 +12,111 @@ public class LevelGenerator : MonoBehaviour {
 	public int width = 10;
 	public GameObject tile = null;
 	public GameObject edge = null;
+	public GameObject edgeDecoration = null;
+	[Range(0f, 1f)]
+	public float edgeScale = 0.5f;
 
 	private GameObject root = null;
+	private float offset;
```
Comment (ratenbuuren) [inline Assets/Scripts/LevelGenerator.cs]: What offset?

File: Assets/Scripts/rotations/RandomRotation.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+public class RandomRotation : Rotation {
+
+  private float interval;
+
+  public RandomRotation(float interval = 360) {
+    this.interval = interval;
+  }
+
+  public override float value() {
+    int n = (int) Math.Ceiling(360 / interval);
```
Comment (ratenbuuren) [inline Assets/Scripts/rotations/RandomRotation.cs]: n? Maybe a more descriptive name

**Review by ratenbuuren — APPROVED** [review]
Feedback incorporated. Looks good for MVP!

**Pull request merged**